src/main/java/org/hiero/sketch/table/Quantiles.java

```diff
@@ -0,0 +1,35 @@
+package org.hiero.sketch.table;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Random;
+
+public class Quantiles {
```

Comment:
Please add a JavaDoc comment describing this class.
Also, I don't think it belongs to the sketch.table; this would rather be in sketch.spreadsheet.